test/functional/fixtures/driver/testcafe-fixtures/prevent-real-action-test.js

```diff
@@ -0,0 +1,11 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones

```

Comment:
there is no need for this comment in this test since we have no a test that should be failed